asciidoc/chapters/assembly_mg_changes-in-http-client.adoc

```diff
@@ -0,0 +1,26 @@
+ifdef::context[:parent-context-of-changes-in-http-client: {context}]
+
+[id="changes-in-http-client_{context}"]
+= Changes in HTTP client
+:context: changes-in-http-client
+
+This section describes the changes in HTTP client.
+
+The following types of {VertX} clients are available:
+
+{VertX} web client::
+Use the {VertX} web client when your applications are HTTP oriented. For example, REST, encoding and decoding HTTP payloads, interpreting the HTTP status response code, and so on.
```

Comment:
@vietj What does "HTTP oriented" mean? Adam K suggested using "web oriented".
Should we elaborate on this?

Reply:
yes that would be better indeed, that's more like a typo.